Pyrado/pyrado/environments/pysim/quanser_cartpole.py

```diff
@@ -341,10 +341,5 @@ def _create_task(self, task_args: dict) -> Task:
         state_des = task_args.get("state_des", np.array([0.0, np.pi, 0.0, 0.0]))
         Q = task_args.get("Q", np.diag([3e-1, 5e-1, 5e-3, 1e-3]))
         R = task_args.get("R", np.diag([1e-3]))
-        rew_fcn = QuadrErrRewFcn(Q, R)
```

Comment:
So you changed the reward function type, but didn't need to change the values of Q and R?